### ISSUE TITLE
Dynamic import detection producing false positives

### ISSUE THREAD

**Issue by jp7837** [pzavolinsky/ts-unused-exports#117]
a.ts:
```
export const A = 1;
export const A_unused = 2;
```
b:ts:
```
import("./a").then(A_imported => {
  console.log(A_imported.A);
});
export const B_unused = 0;
```
tsconfig.json:
```
{
	"compilerOptions": {
		"lib": ["ES2015", "DOM"]
	},
	"files": [
		"a.ts",
		"b.ts"
	]
}
```

**Expected**: A_unused, B_unused marked as unused
**Actual**: A, A_unused, B_unused marked as unused.

A should not be marked as unused since it's referenced in B.

**Comment by jp7837** [comment]
Thanks for fixing @mrseanryan !  May I ask when you think a new version would be published to NPM?

**Comment by mrseanryan** [comment]
np at all - hopefully within next week or so.

**Comment by pzavolinsky** [comment]
@jp7837 @mrseanryan published as `6.1.0`.

Thanks for this contrib!
